src/components/cards/StakingOptionCard.js

```diff
@@ -4,8 +4,8 @@ import * as React from 'react'
 import { View } from 'react-native'
 import FastImage from 'react-native-fast-image'
 
-import { type LiquidityPool } from '../../plugins/stake-plugins/types'
-import { getLiquidityPoolIconUri } from '../../util/CdnUris.js'
+import { type StakeProviderInfo } from '../../plugins/stake-plugins/types.js'
+import { getStakeProviderIcon } from '../../util/CdnUris.js'
```

Comment:
Yeah this was my attempt to genericize our typing a bit more since LPs are specific to EVM Uniswap/Pancakeswap type yield farming. 

Even looking specifically at LPs and their tokens, there's potentially a many-to-one relationship between LP's and reward contracts, so it doesn't make sense to put the concept of the LP into such an outer/top level in our types also for that reason.